test_game.py

```diff
@@ -0,0 +1,150 @@
+"""The test game."""
+
+
+PAYOFF_TABLE = [[2, 3],  # payoff_table[my_move][their_move]
+                [-3, 1]
+               ]
+PLAYER_NUMBER = 2
+ROUND_NUMBER = 5
+
+
+class MoveInfo:
```

Comment:
I believe it's good practice to inherit from object, like `class MoveInfo(object):`

Reply:
That is python 2's new classes - in python 3, all classes are of the new class type (was actually looking at this at work recently)

Reply:
Ah, cool! TIL.